src/ReactWrapper.js

```diff
@@ -438,6 +438,17 @@ export default class ReactWrapper {
   }
 
   /**
+   * Returns a new wrapper with a specific child
+   *
+   * @param {Number} [index]
+   * @returns {ReactWrapper}
+   */
+  childAt(index) {
+    const allChildren = this.flatMap(n => childrenOfInst(n.node));
+    return allChildren.at(index);
```

Comment:
I agree. With a selector this method becomes confusing. Using `.children(selector).at(index)` is more clear at that point.